examples/ui/scroll.rs

```diff
@@ -2,7 +2,7 @@
 
 use bevy::{
     a11y::{
-        accesskit::{NodeBuilder, Role},
+        accesskit::{Node as Accessible, Role},
```

Comment:
This is an unfortunate name conflict (curse `Node` being such a good descriptor for so many things!) Strictly speaking this was an existing issue, since `NodeBuilder` from AccessKit has nothing to do with Bevy's UI node, just made apparent now.

In the examples, you only need to name the AccessKit `Node` in order to construct a Bevy `AccessibilityNode`. Perhaps we could copy the `new()` method into `AccessibilityNode` so end-users can avoid the name conflict?

Reply:
Oh good call. Fixed, thanks for the suggestion.